docs/overrides/assets/css/rzk-render.css

```diff
@@ -1,4 +1,6 @@
 .rzk-render {
   float: right;
   clear: both;
+  position: relative;
+  z-index: 1;
```

Comment:
This is to display SVGs over (not under) code blocks when zoomed in Material theme, if I understood correctly?

Reply:
Yes, exactly (but not just in the Material theme)